tee-worker/omni-executor/parentchain/storage/src/lib.rs

```diff
@@ -1,4 +1,51 @@
 mod account_store;
 pub use account_store::AccountStoreStorage;
 
+use executor_core::storage::Storage;
+use parentchain_rpc_client::{
+	CustomConfig, SubstrateRpcClientFactory, SubxtClient, SubxtClientFactory,
+};
+use parity_scale_codec::Decode;
+use subxt_core::utils::AccountId32 as AccountId;
+
 const STORAGE_PATH: &str = "storage";
+
+pub async fn init_storage(ws_rpc_endpoint: &str) -> Result<(), ()> {
+	let client_factory: SubxtClientFactory<CustomConfig> = SubxtClientFactory::new(ws_rpc_endpoint);
+	let client = client_factory.new_client().await.map_err(|e| {
+		log::error!("Could not create client: {:?}", e);
+	})?;
+
+	init_account_store_storage(&client).await?;
+
+	Ok(())
+}
+
+async fn init_account_store_storage(client: &SubxtClient<CustomConfig>) -> Result<(), ()> {
+	let storage_query = parentchain_api_interface::storage().omni_account().account_store_iter();
```

Comment:
There's no concept of storage proof right? Like we don't have to verify anything that we downloaded

Reply:
Done 👍🏼